stable/README.md

```diff
@@ -126,7 +121,7 @@ oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:nuodb -n nuo
 oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:default -n nuodb
```

Comment:
This is the part I mentioned on the call.  for OCP add-scc-to-user is being added to default as well as nuodb service accounts.  But unless charts are deployed with serviceAccountName set to nuodb, they will install under default and there was no need to specify add-scc-to-user for nuodb.

I think we need to do one or the other.
1) remove default here and explicitly say to set serviceAccountName when using OCP (this can be done thanks to adrians recent change)
2) remove nuodb here and just let the user fallback to the default service account.

Reply:
@acabrele and @vegichan 
I have revised both the K8S and OCP stable page instruction sections based on successful test runs. 
I was not able to reduce OCP to just one service account. I needed both nuodb and default.
Please reopen or comment if there are any further questions. Thanks.

Reply:
@jleslie85 @vegichan @adriansuarez I worked through this with Joe, I think the reason he needed both is because the serviceAccount has been added to the admin and database values, defaulting to nuodb, but THP values did not get updated and so requires the the SCC on default still

Reply:
this doc should be kept up to date with the current state of the code, which is that it requires both. As it was documented.